frame/nomination-pools/test-staking/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
```

Comment:
Why did you opt for a new pallet instead of putting the test in the nomination pool tests?

Reply:
the new pallet has staking as the `StakingInterface`. the nomination pools tests have a mock `StakingInterface` that makes things a bit easier.